app/src/main/java/com/github/polypoly/app/menu/JoinGameLobbyActivity.kt

```diff
@@ -32,28 +32,35 @@ import com.github.polypoly.app.game.GameLobby
 import com.github.polypoly.app.game.user.Skin
 import com.github.polypoly.app.game.user.Stats
 import com.github.polypoly.app.game.user.User
+import com.github.polypoly.app.global.GlobalInstances.Companion.remoteDB
 import com.github.polypoly.app.global.Settings.Companion.DB_GAME_LOBIES_PATH
-import com.github.polypoly.app.network.FakeRemoteStorage
 import com.github.polypoly.app.network.getAllValues
 import com.github.polypoly.app.network.getValue
-import com.github.polypoly.app.ui.theme.PolypolyTheme
 import kotlinx.coroutines.delay
 import timber.log.Timber
+import java.util.concurrent.CompletableFuture
 
 /**
  * Activity where the user can join a gameLobby
  */
-class JoinGameLobbyActivity : ComponentActivity() {
+class JoinGameLobbyActivity : MenuActivity("Join a game") {
+    companion object {
+        const val POLLING_INTERVAL = 5000L
+    }
 
     /**
      * The attributes of the class
      */
     private var gameLobbyCode: String = ""
 
+    private val fakeAuthenticatedUser = User(7, "fake_user", "I am fake until google authentication is setup", Skin(0, 0, 0),
+        Stats(0, 0, 0, 0, 0), listOf(), mutableListOf()
+    )
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContent {
-            PolypolyTheme {
+            MenuContent{
```

Comment:
Once again, let's stick to the convention of having a composable that contains all the UI named `ActivityNameContent` so that the onCreate function isn't that big.

Reply:
This will be part of my undone task that I will carry over next sprint, but thanks for the reminder!